src/components/CoreApps/index.js

```diff
@@ -0,0 +1,88 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import React from 'react'
+import getLatestVersion from '../../get-latest-version'
+import AppList from '../AppList'
+
+const coreAppNames = [
+    'App Management',
+    'Cache Cleaner',
+    'Capture',
+    'Dashboard',
+    'Data Visualizer',
+    'Datastore Management',
+    'Import/Export',
+    'Interpretations',
+    'Maintenance',
+    'Menu Management',
+    'Messaging',
+    'Reports',
+    'SMS Configuration',
+    'Scheduler',
+    'Settings',
+    'Translations',
+    'User Management',
+]
+
+const query = {
+    coreApps: {
+        resource: 'apps',
+        params: {
+            bundled: true,
+        },
+    },
+    appHub: {
+        resource: 'appHub/v1/apps',
+    },
+}
+
+const CoreApps = () => {
+    const { loading, error, data } = useDataQuery(query)
+
+    const overridenCoreApps = data?.coreApps.filter(app => app.bundled)
+    const apps = coreAppNames
+        .map(coreAppName => {
+            const overridenApp = overridenCoreApps?.find(
+                a => a.name === coreAppName
+            )
+            if (overridenApp) {
+                return overridenApp
+            }
+            return {
+                name: coreAppName,
+                bundled: true,
+                short_name: coreAppName,
+                developer: {
+                    company: 'DHIS2',
+                },
+                icons: {},
```

Comment:
This might be OK for the first pass but is not ideal - this really should show the icon for the bundled application, which should be available at `/icons/dhis-web-${short_name}.png` and is definitely included in the getModules.action response (ugly I know, but it's what we have)